resources/migrations/V1.0.2__Backups-UnsafeSink.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE backups
+    ADD sink_is_immutable BOOLEAN DEFAULT FALSE NOT NULL;
```

Comment:
i can imagine that a timestamp of last check and last transition would be helpful for auditing this column. What is your standpoint on this? Also there is no representation on the status if the actual check in newCheckImmutableBackupsService failed due to processing errors

Reply:
after discussion in blocker after Kaan was gone: 
- create a new table for sink_project with the attributes
- backup_only_services
- single_writer_iam